Http/Http.cs

```diff
@@ -0,0 +1,352 @@
+		{
+			return superHeaders.Remove(key);
+		}
+
+		#region Creation Helpers
+
+		/// <summary>
+		///   <para>Creates a HttpRequest configured for HTTP GET.</para>
+		/// </summary>
+		/// <param name="uri">The URI of the resource to retrieve via HTTP GET.</param>
+		/// <returns>
+		///   <para>A HttpRequest object configured to retrieve data from uri.</para>
+		/// </returns>
+		public static HttpRequest Get(string uri)
+		{
+			return new HttpRequest(UnityWebRequest.Get(uri));
+		}
+
+		/// <summary>
+		///   <para>Create a HttpRequest configured to send form data to a server via HTTP POST.</para>
+		/// </summary>
+		/// <param name="uri">The target URI to which form data will be transmitted.</param>
+		/// <param name="postData">Form body data. Will be URLEncoded via WWWTranscoder.URLEncode prior to transmission.</param>
+		/// <returns>
+		///   <para>A HttpRequest configured to send form data to uri via POST.</para>
+		/// </returns>
+		public static HttpRequest Post(string uri, string postData)
+		{
+			return new HttpRequest(UnityWebRequest.Post(uri, postData));
+		}
+
+		/// <summary>
+		///   <para>Create a HttpRequest configured to send json data to a server via HTTP POST.</para>
+		/// </summary>
+		/// <param name="uri">The target URI to which json data will be transmitted.</param>
+		/// <param name="json">Json body data.</param>
+		/// <returns>
+		///   <para>A HttpRequest configured to send json data to uri via POST.</para>
+		/// </returns>
+		public static HttpRequest PostAsJson(string uri, string json)
+		{
+			var unityWebRequest = new UnityWebRequest(uri, UnityWebRequest.kHttpVerbPOST)
+			{
+				uploadHandler = new UploadHandlerRaw(Encoding.UTF8.GetBytes(json))
+				{
+					contentType = "application/json"
+				},
+				downloadHandler = new DownloadHandlerBuffer()
+			};
+			return new HttpRequest(unityWebRequest);
+		}
+
+		/// <summary>
+		///   <para>Create a HttpRequest configured to send json data to a server via HTTP POST.</para>
+		/// </summary>
+		/// <param name="uri">The target URI to which json data will be transmitted.</param>
+		/// <param name="payload">The object to be parsed to json data.</param>
+		/// <returns>
+		///   <para>A HttpRequest configured to send json data to uri via POST.</para>
+		/// </returns>
+		public static HttpRequest PostAsJson<T>(string uri, T payload) where T : class
+		{
+			return PostAsJson(uri, JsonUtility.ToJson(payload));
+		}
+
+		/// <summary>
+		///   <para>Create a HttpRequest configured to send form data to a server via HTTP POST.</para>
+		/// </summary>
+		/// <param name="uri">The target URI to which form data will be transmitted.</param>
+		/// <param name="formData">Form fields or files encapsulated in a WWWForm object, for formatting and transmission to the remote server.</param>
+		/// <returns>
+		///   <para>A HttpRequest configured to send form data to uri via POST.</para>
+		/// </returns>
+		public static HttpRequest Post(string uri, WWWForm formData)
```

Comment:
In the `PostAsJson` version you override the header to be `application/json`. Does Unity automatically set the `x-www-form-urlencoded` header for you?

Reply:
Yes, it does